scripts/netContain/Dockerfile

```diff
@@ -22,7 +22,7 @@ FROM ubuntu:16.04
 # ENV export https_proxy=https://proxy.localhost.com:8080
 
 RUN apt-get update \
- && apt-get install -y openvswitch-switch=2.5.0-0ubuntu1 \
+ && apt-get install -y openvswitch-switch=2.5* \
```

Comment:
1. Yes, we need to change this in ansible as well.
2. This is a Dockerfile for a Docker image. We use this for our netplugin Docker images.
3. I don't think that's the case for now. This OVS version seems fine for now. We can revisit this at any point in the future.